packages/lib/markdownAndSanitizeClientSide.ts

```diff
@@ -0,0 +1,21 @@
+import DOMPurify from "dompurify";
```

Comment:
I think we can remove this duplicate by call markdownAndSanitize in tRPC calls and avoiding to do it on client side.